src/containers/Bounty/index.js

```diff
@@ -167,7 +165,7 @@ class BountyComponent extends React.Component {
             iconColor="red"
             title="Could not find that bounty"
             text="Try refreshing, or make sure your url is correct"
-            icon={['far', 'exclamation-triangle']}
+            icon={['fal', 'exclamation-triangle']}
```

Comment:
Did yo umea . to change this from far to fal?

Reply:
yes, there was an issue where the far one didn't exist

Reply:
Ah maybe check with Corwin which one he wants. If he wants the FAR - then just import that.

Reply:
I believe zero states and errors are using light, so this looks right